aten/src/ATen/native/MaxPooling.cpp

```diff
@@ -4,23 +4,20 @@
 #include <ATen/native/DispatchStub.h>
 #include <ATen/native/MaxPooling.h>
 #include <ATen/native/Pool.h>
+#include <iostream>
```

Comment:
nit: probably we don't need it, (might have lost after debugging locally).

Reply:
Oops. Thanks!